UNPRIVILEGED_INSTALL.md

```diff
@@ -0,0 +1,210 @@
+# Unprivileged Installation
```

Comment:
This is great! Def a necessary inclusion

Reply:
Might make note that these steps need to be done by an administrator "because they require root"
If they did not then cp-ansible could do them

Reply:
Thanks, I'll probably update after the test session that I'm starting today on a fresh system.

Reply:
yes, this is true